apps/passport-server/src/services/telemetryService.ts

```diff
@@ -66,6 +66,7 @@ export async function writeMarker(
   }
 
   try {
+    // eslint-disable-next-line no-restricted-globals
```

Comment:
may be a dumb question, but why don't we retry honeycomb?

Reply:
- honeycomb has not historically caused any issues
- plugging it into `instrumentedFetch`, which is where i've implemented the retry logic, would cause a ciruclar sort of thing because the instrumentation IS honeycomb.